src/common/components/BaniList/baniOrderHelper.js

```diff
@@ -0,0 +1,23 @@
+const oderedBani = (baniList) => {
+  const defaultBaniOrder = require("../../defaultBaniOrder.json");
+  const banis = [];
+
+  defaultBaniOrder.baniOrder.forEach((obj) => {
+    if (obj.id) {
+      const baniItem = baniList.filter((item) => item.id === obj.id)[0];
```

Comment:
This code is repetitive. Would it work if you write line 7 and line 14 just once on line 6 above the if statement?

```js
line:7 const baniItem = baniList.filter((item) => item.id === obj.id)[0];
line:14 const bani = baniList.filter((listItem) => listItem.id === item.id)[0];
```

Reply:
fixed